lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1786,6 +1786,16 @@ defmodule Kernel.SpecialForms do
 
       &local_function/1
 
+  Note that `&local_function/1` creates a local capture, but
+  `&__MODULE__.local_function/1` or `&imported_function/1` create a remote
+  capture. This has performance implications: remote captures are faster than
```

Comment:
Perhaps we should be careful not to over-emphasize this, as it should be a very small overhead (re-linking the [benchmark](https://github.com/sabiwara/elixir_benches/commit/9b721f3b19f8af9d9d63a30618cdafeaa6367883#diff-1d96315ec16e4339c58b7f6528ee1f17decb95fbc30e301d6eb1cedfd9bdd1e6R27) I made back then)
```suggestion
  capture. This has performance implications: remote captures are slightly faster than
```